NEWS.md

```diff
@@ -1,30 +1,28 @@
 
-# renv 0.18.0  (UNRELEASED)
+# renv 1.0.0
```

Comment:
Do you want me to take a pass through the news entries? We usually consolidate into "new features", "lifecycle changes", and "bug fixes and minor improvements" and then alphabetise by primary function within those sections.

Reply:
Sure, that would be appreciated.